docs/source/simulators/CircuitSimulator.rst

```diff
@@ -1,5 +1,42 @@
-CircuitSimulator
-================
+Circuit Simulator
+=================
+
+The *Circuit Simulator* is the "default" simulator in DDSIM.
+It can handle all input supplied as :code:`QuantumComputation` from the QFR. This includes, but is not limited to,
+OpenQASM and real files as well as pre-defined algorithm like QFT, Grover, and random clifford circuits.
+
+Capabilities:
+
+- Exact Simulation :cite:p:`zulehner2019advanced,DBLP:conf/iccad/ZulehnerHW19`
+- Weak Simulation :cite:p:`DBLP:conf/dac/HillmichMW20`
+- Approximate Simulation with different strategies :cite:p:`10.1145/3530776,DBLP:conf/date/HillmichKMW21`
+
+    - Fidelity-driven
+    - Memory-driven
+
+
+Usage in Python
+###############
+
+Available backends in the provider :code:`DDSIMProvider`:
+
+- :code:`qasm_simulator` to simulate and sample from the resulting state
+- :code:`statevector_simulator` to simulate and explicitly return the state vector (which will require an exponential amount of memory in the number of qubits)
+
+Using the Circuit Simulator without Qiskit is also possible by passing a file name like in the following example:
+
+.. code-block:: python
+
+    from mqt import ddsim
+
+    sim = ddsim.CircuitSimulator("ghz_03.qasm")
+    result = sim.simulate(1000)  # 1000 shots
+    # results will contain a dictional like {'000': 510, '111': 490}
+
+This of course assumes a file :code:`ghz_03.qasm` exists.
```

Comment:
I am fine with this for now, but I would like to have two points addressed in a future revision:
 1) the standalone simulator can still take Qiskit circuits as input and that is actually the more convenient thing to do and probably worth showing off first/predominantly.
 2) all the other simulators can also be constructed and used like this. This should be reflected in the documentation of the individual simulators.